internal/livelint/mock_ui_test.go

```diff
@@ -0,0 +1,331 @@
+// Code generated by moq; DO NOT EDIT.
```

Comment:
Shold we move these into a separate `mock` package, so they don't clutter this space?

Reply:
Will post-pone this refactoring as we are doing a major refactoring at the moment.